projects/1.16.1/assets/atmospheric/atmospheric/lang/zh_cn.json

```
@@ -0,0 +1,430 @@
+	
+	"block.atmospheric.ivory_fumarole_block": "象牙白气孔方块",
+	"block.atmospheric.peach_fumarole_block": "桃色气孔方块",	
+	"block.atmospheric.persimmon_fumarole_block": "柿子色气孔方块",
+	"block.atmospheric.saffron_fumarole_block": "藏红气孔方块",	
+	
+	"block.atmospheric.ivory_travertine_vertical_slab": "象牙白石灰华竖直台阶",
+	"block.atmospheric.peach_travertine_vertical_slab": "桃色石灰华竖直台阶",	
+	"block.atmospheric.persimmon_travertine_vertical_slab": "柿子色石灰华竖直台阶",
+	"block.atmospheric.saffron_travertine_vertical_slab": "藏红石灰华竖直台阶",	
+	
+	"item.atmospheric.spring_water": "泉水",
+	"item.atmospheric.combustive_tonic": "易燃的奎宁水",
+	"item.atmospheric.charged_tonic": "带电的奎宁水",
+
+	"biome.atmospheric.hot_springs": "热泉",
+	
+	"death.attack.fumarole": "%1$s试图靠近喷泉但被轰开了",
+	"death.attack.fumarole.player": "%1$s在逃离%2$s时试图靠近喷泉",
+	
+  "_comment3": "START OF THE DUNES SECTION",
+  
+	"block.atmospheric.yucca_log": "丝兰原木",
+	"block.atmospheric.yucca_wood": "丝兰木",
+	"block.atmospheric.stripped_yucca_log": "去皮丝兰原木",
+	"block.atmospheric.stripped_yucca_wood": "去皮丝兰木",
+	"block.atmospheric.yucca_planks": "丝兰木板",
+	"block.atmospheric.yucca_stairs": "丝兰木楼梯",
+	"block.atmospheric.yucca_slab": "丝兰木台阶",
+	"block.atmospheric.yucca_pressure_plate": "丝兰木压力板",
+	"block.atmospheric.yucca_button": "丝兰木按钮",
+	"block.atmospheric.yucca_door": "丝兰木门",
+	"block.atmospheric.yucca_trapdoor": "丝兰木活板门",
+	"block.atmospheric.yucca_fence": "丝兰木栅栏",
+	"block.atmospheric.yucca_fence_gate": "丝兰木栅栏门",
+	"block.atmospheric.yucca_sapling": "丝兰树苗",
+	"block.atmospheric.yucca_leaves": "丝兰树叶",
+	"block.atmospheric.yucca_sign": "丝兰木告示牌",
+	"block.atmospheric.yucca_wall_sign": "墙上的丝兰木告示牌",
+	
+  "block.atmospheric.yucca_bookshelf": "丝兰木书架",
+  "block.atmospheric.yucca_vertical_slab": "丝兰木竖直台阶",
+  "block.atmospheric.vertical_yucca_planks": "竖直丝兰木板",
+  "block.atmospheric.yucca_ladder": "丝兰木梯子",
+  "block.atmospheric.yucca_leaf_carpet": "丝兰树叶地毯",
+  "block.atmospheric.yucca_chest": "丝兰木箱子",
+  "block.atmospheric.yucca_trapped_chest": "丝兰木陷阱箱",
+  "block.atmospheric.yucca_beehive": "丝兰木蜂箱",
+  
+	"block.atmospheric.yucca_flower": "丝兰花",
+	"block.atmospheric.tall_yucca_flower": "高丝兰花",
+	"block.atmospheric.yucca_branch": "丝兰树枝",	
+	"block.atmospheric.yucca_bundle": "丝兰果捆",	
+	"block.atmospheric.roasted_yucca_bundle": "烤丝兰果捆",	
+	
+	"block.atmospheric.potted_yucca_sapling": "丝兰树苗盆栽",
+	"block.atmospheric.potted_yucca_flower": "丝兰花盆栽",
+	"block.atmospheric.potted_barrel_cactus": "桶仙人掌盆栽",
+	"block.atmospheric.potted_gilia": "吉莉草盆栽",
+	
+	"block.atmospheric.yucca_cask": "桶装丝兰果",
+	"block.atmospheric.roasted_yucca_cask": "桶装烤丝兰果",
+	"block.atmospheric.yucca_gateau": "丝兰果大蛋糕",
+	
+	"block.atmospheric.barrel_cactus": "桶仙人掌",
+	"block.atmospheric.barrel_cactus_batch": "一捆桶仙人掌",
+	"block.atmospheric.aloe_vera": "芦荟",
+	"block.atmospheric.tall_aloe_vera": "芦荟",
+	"block.atmospheric.aloe_bundle": "芦荟捆",
+	"block.atmospheric.aloe_gel_block": "芦荟凝胶块",
+	"block.atmospheric.gilia": "吉莉草",
+	
+	"block.atmospheric.arid_sand": "干燥沙子",

```

```suggestion
	"block.atmospheric.arid_sand": "干旱沙子",

```
**干燥**一般形容气候，此处用**干旱**会比较合适，下同。

---

已修正

---

> 干燥(drying)是利用热能使湿物料中的湿分(水分或其他溶剂)气化，并利用气流或真空带走气化了的湿分，从而获得干燥物料的操作。如湿法制粒中物料的干燥、溶液的喷雾干燥、流浸膏的干燥等。

> 干旱是指淡水总量少，不足以满足人的生存和经济发展的气候现象，一般是长期的现象。

完全反了，这里指的是单个方块的湿润程度，使用干燥合情合理，干旱是气候用语。

---

。。。好吧